simularium/simularium-website
# Feature/place visual glossary key next to image on wider screens

Problem
=======
It's hard to look at the visual glossary image and its legend at the same time with the current layout.

<img width="2070" alt="image" src="https://user-images.githubusercontent.com/12690133/153691437-a71a16ac-be45-45c3-87b5-df8ed339949d.png">


Closes #167 

Solution
========
* Constrain the legend box's height and add a scroll bar inside it
* Place the image and the legend side-by-side when the browser window is wide enough
* Place the legend below the image when the browser window is not wide enough

#### Details
* The above layouts switch back and forth dynamically as the user resizes the window
* On page load, if the screen is not wide enough for side-by-side, the legend box is collapsed by default so that the user can see there is more content below the fold
* When the screen is wide enough and the image and legend are side-by-side, the legend box is always in an open state (can't collapse it)

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Pull this branch and `npm start`
2. Go to Help -> Quick start
3. Check out the visual glossary, try resizing your window, and refreshing the page at different widths

Screenshots (optional):
-----------------------
### Wide screen
<img width="1809" alt="image" src="https://user-images.githubusercontent.com/12690133/153691669-d0da7cd5-6a33-4cb3-954a-24c830dbdb1a.png">

### Narrower screen
<img width="600" alt="image" src="https://user-images.githubusercontent.com/12690133/153691722-3e153ead-554e-4c25-95d9-7ac0d14c8da5.png">

<img width="600" alt="image" src="https://user-images.githubusercontent.com/12690133/153691715-b44a8b6d-9c44-4792-bcbd-003885948cfe.png">
